test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -354,13 +354,14 @@ UCS_TEST_P(test_ucp_mmap, alloc) {
 }
 
 UCS_TEST_P(test_ucp_mmap, alloc_mem_type) {
-    std::vector<ucs_memory_type_t> mem_types = mem_buffer::supported_mem_types();
+    const std::vector<ucs_memory_type_t> &mem_types =
+            mem_buffer::supported_mem_types();
     ucs_status_t status;
     bool is_dummy;
     bool expect_rma_offload;
 
-    for (std::vector<ucs_memory_type_t>::iterator mem_type = mem_types.begin();
-         mem_type != mem_types.end(); ++mem_type) {
+    for (std::vector<ucs_memory_type_t>::const_iterator mem_type = mem_types.cbegin();
```

Comment:
maybe use c++11 for loop, now that we allow it in gtest?

Reply:
done, now it is much easier